types/tizen-common-web/index.d.ts

```diff
@@ -1,84 +1,14699 @@
+     * @throw WebAPIException with error type UnknownError in an unspecified error case.
+     *
+     * @privilegeLevel 5.0 public
+     * @privilegeName 5.0 http://tizen.org/privilege/tv.audio
+     * @warning 5.0 http://tizen.org/privilege/tv.audio (public level) has been deprecated since 5.0.
+     */
+    setVolumeUp(): void;
+    /**
+     * Unregisters the volume change callback for detecting the volume changes.
+     *
+     * Calling this function has no effect if listener is not set.
+     *
+     * @throw WebAPIException with error type SecurityError, if the application does not have the privilege to call this method.
+     * This type of error is deprecated since Tizen 5.0.
+     * @throw WebAPIException with error type UnknownError in an unspecified error case.
+     *
+     * @privilegeLevel 5.0 public
+     * @privilegeName 5.0 http://tizen.org/privilege/tv.audio
+     * @warning 5.0 http://tizen.org/privilege/tv.audio (public level) has been deprecated since 5.0.
+     */
+    unsetVolumeChangeListener(): void;
+}
+/**
+ * Key-value pair container.
+ *
+ * Bundle keys are always strings.
+ * All supported value types are specified in the BundleValueType enum.
+ *
+ * Plain dictionary will be implicitly converted to the Bundle object in every
+ * Bundle context within WebAPI.
+ *
+ * @since 5.5
+ */
+export class Bundle {
+    constructor(json?: any);
+    /**
+     * Calls the callback function for each item stored in the bundle.
+     *
+     * If bundle is empty the callback function will not be called.
+     *
+     * @param callback Function to be called for each entry.
+     *
+     * @remark Empty arrays are treated like string arrays.
+     */
+    forEach(callback: BundleItemCallback): void;
+    /**
+     * Gets value stored under given key.
+     *
+     * @param key Bundle entry key.
+     *
+     * @returns Bundle entry value for a given key.
+     *
+     * @throw WebAPIException with error type NotFoundError, if the key could not be found.
+     */
+    get(key: string): any;
+    /**
+     * Inserts the key-value pair.
+     *
+     * @param key Entry key.
+     * @param value Entry value.
+     *
+     * @remark Any value type not specified in the BundleValueType enum will be converted to a string.
+     * @remark Empty array value will be treated like STRING\_ARRAY.
+     */
+    set(key: string, value: any): void;
+    /**
+     * Converts bundle to JSON-compatible object.
+     *
+     * @returns JSON-compatible object.
+     */
+    toJSON(): any;
+    /**
+     * Gets type of the value for a given key.
+     *
+     * @param key Entry key.
+     *
+     * @returns Entry value type.
+     *
+     * @throw WebAPIException with error type NotFoundError, if the key could not be found.
+     *
+     * @remark If the value for the given key is an empty array this function returns STRING\_ARRAY.
+     */
+    typeOf(key: string): BundleValueType;
+}
+export interface Bundle {
```

Comment:
Another case of incorrectly declaring the *static* (i.e., constructor) side of the class on an interface that represents the *instance*. Please remove.